plugins/destination/postgresql/client/migrate.go

```diff
@@ -377,6 +374,29 @@ func (c *Client) createTableIfNotExist(ctx context.Context, table *schema.Table)
 	return nil
 }
 
+func (c *Client) listPKs(table *schema.Table) []string {
+	// If PKS are not needed for the writeMode then just return early
+	if !c.enabledPks() {
+		return []string{}
+	}
+	pks := []string{}
+	if c.spec.PKMode == specs.PKModeCQID {
+		pks = append(pks, schema.CqIDColumn.Name)
```

Comment:
I'm not sure destinations should reference CqID directly. Update the table PKs detentions in the SDK based on the spec PK mode, WDYT?

Reply:
If we don't reference the `CqID` how do we enable users to be able to switch between `composite-keys` and `cq-ids`?

Reply:
See my comment here https://github.com/cloudquery/plugin-sdk/pull/732#pullrequestreview-1339843474
We can update the column options here https://github.com/cloudquery/plugin-sdk/blob/ac55bec5f3c06b3cdd9a0409394d5bf5cf059137/plugins/source/plugin.go#L282 for example

Reply:
If we do it on destinations, we can override the PK definitions here https://github.com/cloudquery/plugin-sdk/blob/ac55bec5f3c06b3cdd9a0409394d5bf5cf059137/plugins/destination/plugin.go#L187

Reply:
Great call! Thanks for the heads up